### ISSUE TITLE
Can't use the world `Block` anywhere in test file

### ISSUE THREAD

**Issue by thdxr** [aelbore/esbuild-jest#72]
I was getting strange compilation errors in ESM and finally narrowed it down to my usage of the word `Block` anywhere in source code.  This included comments, tried moving it around everywhere as well.

Eventually found this:
https://github.com/aelbore/esbuild-jest/blob/master/src/index.ts#L33

So...guess that TODO has come home to roost

**Comment by andrewbranch** [comment]
I think I just hit this too. 😬

**Comment by ghost** [comment]
I encountered the same issue with a function called `clock`.

**Comment by sebastianrothe** [comment]
The problem is the word `ock`, it is parsed and interpreted somewhere.

**Comment by bhousel** [comment]
yep, see https://github.com/aelbore/esbuild-jest/pull/44#issuecomment-893481820

**Comment by gwsbhqt** [comment]
Does anyone know the historical reason for this? What a stupid decision it seems now. It killed my whole day

**Comment by daveclay** [comment]
How does this issue exist, and how is it not even acknowledged? Is the esbuild-jest community _just not using mocks_? How did someone think https://github.com/aelbore/esbuild-jest/blob/master/src/index.ts#L33 was _acceptable_?
